data.sql

```diff
@@ -8,3 +8,68 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', 'Jan 7, 2021', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', 'May 12, 2017', 5, true, 11);
+
+/* Project - query and update animals table */
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES ('Charmander', 'Feb 8, 2020', 0, false, 11);
+
+INSERT INTO animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES ('Plantmon', 'Nov 15, 2021', 2, true, 5.7);
```

Comment:
- According to the project requirements, the weight here should be `-5.7kg`. Please fix this 🙏